GetIntoTeachingApi/Controllers/GetIntoTeaching/TeachingEventsController.cs

```diff
@@ -78,13 +78,19 @@ public class TeachingEventsController : ControllerBase
 
             var teachingEvents = await _store.SearchTeachingEventsAsync(request);
 
+            // TODO: Fix. Prometheus can only be passed two labels currently. Don't know how we would want to handle multiple type IDs.
```

Comment:
You can pass more than two but you need to define them explicitly on the metric here:

https://github.com/DFE-Digital/get-into-teaching-api/blob/master/GetIntoTeachingApi/Services/MetricService.cs#L23

I would rename the `type_id` metric to `type_ids` and pass it a comma separated list in this case (it would be more flexible if we had a label for each type id but it uses a bunch more space and we don't need that for what we're reporting on here)